test/test_cuda.py

```diff
@@ -462,7 +462,7 @@ def test_specify_improper_device_name(self):
                 os.remove(fname)
 
     def test_get_device_index(self):
-        from torch.cuda._utils import _get_device_index
+        from torch._utils import _get_device_index
```

Comment:
As we discussed offline, we do not need to change it now.

Reply:
done.